### ISSUE TITLE
MultiMap improvements

### ISSUE THREAD

**Issue by alexed1** [alexed1/LightningFlowComponents#510]
@NickSauer7 What's the status of that nice multimap extension? I see you have a draft blog post on unofficialsf.com. is that ready to go live? 

how about packaging the extension? If you want to add this code as a pull request to your existing map component on  FlowComponents, I'm happy to generate a package for it...

**Comment by NickSauer7** [comment]
Hey Alex.  That blog post is the one that corresponds to the multiMapFSC component that you already have in FlowComponents.  
The blog post itself just never got published, which I realized last week.  I did a post on LinkedIn pointing to it, with description on my blog, and seems to have some level of very unexpected traffic. 
 I guess we could still publish the draft blog post so there is some summary documentation around it on the site.  Whatever you think is best - I just didn’t want to bug you on it as seemed more of a conversion from aura->LWC with a couple updates as opposed to something of real interest.  I think maybe folks didn’t realize the other aura component version was there.
Best regards,Nick


Sent from Yahoo Mail for iPhone


On Saturday, October 10, 2020, 8:19 PM, Alex Edelstein <notifications@github.com> wrote:




@NickSauer7 What's the status of that nice multimap extension? I see you have a draft blog post on unofficialsf.com. is that ready to go live?

how about packaging the extension? If you want to add this code as a pull request to your existing map component on FlowComponents, I'm happy to generate a package for it...
